apps/spruce/src/pages/adminSettings/tabs/GeneralTab/ExternalCommunicationsTab/schemaFields.ts

```diff
@@ -0,0 +1,217 @@
+import widgets from "components/SpruceForm/Widgets";
+import { PriorityLevel } from "gql/generated/types";
+import { nestedObjectGridCss, fullWidthCss } from "../../sharedStyles";
+
+export const jira = {
+  schema: {
+    email: {
+      type: "string" as const,
+      title: "Email",
+    },
+    host: {
+      type: "string" as const,
+      title: "Host",
+    },
+    personalAccessToken: {
+      type: "string" as const,
+      title: "Personal Access Token",
+    },
+  },
+  uiSchema: {
+    "ui:fieldCss": nestedObjectGridCss,
+    personalAccessToken: {
+      "ui:widget": widgets.TextWidget,
+      "ui:options": {
+        inputType: "password",
+      },
+    },
+  },
+};
+
+export const slack = {
+  schema: {
+    token: {
+      type: "string" as const,
+      title: "Token",
+    },
+    name: {
+      type: "string" as const,
+      title: "App Name",
+    },
+    level: {
+      type: "string" as const,
+      title: "Priority Level",
+      enum: Object.values(PriorityLevel),
+      default: "",
+    },
+    channel: {
+      type: "string" as const,
+      title: "Channel",
+    },
+    hostname: {
+      type: "string" as const,
+      title: "Hostname",
+    },
+    optionsName: {
+      type: "string" as const,
+      title: "Name",
+    },
+    username: {
+      type: "string" as const,
+      title: "Username",
+    },
+    fieldsSet: {
+      type: "array" as const,
+      title: "Fields Set",
+      default: [],
+      items: {
+        type: "string" as const,
+        properties: {
+          value: {
+            type: "string" as const,
+          },
+        },
+      },
+    },
+    basicMetadata: {
+      type: "boolean" as const,
+      title: "Basic Metadata",
```

Comment:
This is the text from prod
```suggestion
      title: "Add basic metadata",
```